src/etc/inc/interfaces.inc

```diff
@@ -3868,8 +3868,9 @@ function interface_dhcpv6_configure($interface = "wan", $wancfg) {
 	$wanif = get_real_interface($interface, "inet6");
 	$dhcp6cconf = "";
 
-	if (isset($wancfg['dhcp6-duid'])) {
-	// Write the DUID file
+	if (isset($wancfg['dhcp6-duid']) &&
+		!empty($config['system']['global-v6duid'])) { 
```

Comment:
On multiline statements, second line should be indented using 4 spaces after previous line indent